llvm/test/CodeGen/RISCV/cfi-multiple-locations.mir

```diff
@@ -1,6 +1,7 @@
 # RUN: not --crash llc %s -mtriple=riscv64 \
 # RUN: -run-pass=cfi-instr-inserter \
-# RUN: -riscv-enable-cfi-instr-inserter=true 2>&1 | FileCheck %s
+# RUN: -riscv-enable-cfi-instr-inserter=true \
+# RUN: -o - 2>&1 | FileCheck %s
```

Comment:
If you don't want to check stdout I think you do `-o /dev/null`, just in case it messes with the stderr check lines.